packages/next/src/build/analysis/get-page-static-info.ts

```diff
@@ -51,14 +51,14 @@ export interface PageStaticInfo {
 const CLIENT_MODULE_LABEL =
   /\/\* __next_internal_client_entry_do_not_use__ ([^ ]*) (cjs|auto) \*\//
 const ACTION_MODULE_LABEL =
-  /\/\* __next_internal_action_entry_do_not_use__ ([^ ]+) \*\//
+  /\/\* __next_internal_action_entry_do_not_use__ '([^']+)'([^ ]+) \*\//
```

Comment:
```suggestion
  /\/\* __next_internal_action_entry_do_not_use__ '([^']+)' ([^ ]+) \*\//
```

Or am I missing anything?